router/main.py

```
@@ -2697,6 +2676,10 @@ async def stream_generator():
                                 _close_prop_ctx(_litellm_gen_prop)
                                 finalized = True
                             except Exception as ex:
+                                if hasattr(ex, "status_code") and getattr(ex, "status_code") == 429:
```

Ruff (0.15.21)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)